client/src/components/molecules/EditForm/ItemCreateForm.js

```diff
@@ -39,10 +39,25 @@ export const ItemCreateForm = (data) => {
     setShowBatchOptions(checked);
   };
 
+  const sortQuantities = (formValues) => {
+    const fieldName =
+      formValues.shoeSizes.length > 0 ? 'shoeSizes' : 'clothingSizes';
+    const sizeOrder =
+      formValues.shoeSizes.length > 0 ? shoeSizeOptions : clothingSizeOptions;
+    const quantities = formValues[fieldName];
+    const keyValueArray = Object.entries(quantities);
+    keyValueArray.sort(
+      (a, b) => sizeOrder.indexOf(a[0]) - sizeOrder.indexOf(b[0])
+    );
+    const sortedQuantities = Object.fromEntries(keyValueArray);
+    return { ...formValues, [fieldName]: sortedQuantities };
+  };
```

Comment:
good idea!

Reply:
Just being lazy and want to make sure this sorting will be happening not only on the item creation but also on item updates??

Reply:
A yikes, a very good point that I overlooked. 

I'll push the changes in for now (as they now contain some fixes that I'd like to get re-tested), and then will address this in the next PR :)